unified-scheduler-pool/src/lib.rs

```diff
@@ -1957,9 +1961,14 @@ impl<S: SpawnableScheduler<TH>, TH: TaskHandler> ThreadManager<S, TH> {
             }
         };
 
+        let mode_char = match scheduling_mode {
+            BlockVerification => 'V',
+            BlockProduction => 'P',
+        };
```

Comment:
Can we do "BVer" and "BProd"?

Reply:
both thread names are quite close full to the max thread name of 15 chars..:

```
123456789012345
solScheduleX
solScHandleXNN
```

Reply:
ah I miscounted by 1, but `solScheduleBPrd` is the max length. and there should only be 1